services/frontend/src/components/profileForms/careerManagementForm/qualifiedPoolsForm/QualifiedPoolsFormView.scss

```diff
@@ -0,0 +1,22 @@
+.entryTitle {
+  font-size: 0.85em !important;
+}
+
+.formItem {
+  font-size: 1.2em !important;
+  margin: 0;
+}
+
+.topRow {
+  background-color: #dfe5e4;
+  padding: 15px 10px 15px 10px;
+  margin-bottom: 17px;
+}
+
+.formOutlined {
+  margin-right: 0.5em;
+}
+
+.deleteButton {
+  float: right;
+}
```

Comment:
for these class names, (haven't tested yet), but maybe you need to prepend something unique from the component, since the styling is general and may overlap - may not be applicable here, but maybe